File: 프로그래머스/lv0/120895. 인덱스 바꾸기/인덱스 바꾸기.js
function solution(my_string, num1, num2) {
    // 방법 2 - 리스트로 바꾼 후 글자 바꾸기, 구조분해할당 사용
    let arr = my_string.split('');
    [arr[num1], arr[num2]] = [arr[num2], arr[num1]];
    return arr.join('');
    
//     // 방법 1 - 리스트로 바꾼 후 글자 바꾸기
//     const L = [...my_string]
//     const tem = L[num1]
    
//     L[num1] = L[num2]
//     L[num2] = tem
    
//     return L.join('')
}

File: 프로그래머스/lv0/120895. 인덱스 바꾸기/인덱스 바꾸기.py
def solution(my_string, num1, num2):
    # 문자열을 리스트로 만든 후 글자 바꾸기
    L = list(my_string)
    L[num1], L[num2] = L[num2], L[num1]
    return ''.join(L)